routes/projectsDB.ts

```diff
@@ -73,6 +74,7 @@ async function startServer() {
       VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)`;
         try {
             queryDatabase(client, query, values);
+            synchronizeLocal(client);
```

Comment:
there's an inconsistency in synchronous actions here. queryDatabase is synchronous but client.query is not awaited despite it being awaited in documentation. synchronizeLocal is declared as an async function but is not awaited.